src/Evolution/Systems/Cce/AnalyticSolutions/SphericalMetricData.cpp

```diff
@@ -47,7 +47,7 @@ void SphericalMetricData::variables_impl(
       get<Tags::detail::InverseCartesianToSphericalJacobian>(
           intermediate_variables);
 
-  spherical_metric(make_not_null(&intermediate_spherical_metric), time);
+  spherical_metric(make_not_null(&intermediate_spherical_metric), l_max, time);
```

Comment:
The pup function for the `SphericalMetricData` class is tested indirectly by the test of the serialization of `LinearizedBondiSachs` (and will be also tested by other derived classes), but isn't tested directly because `SphericalMetricData` is abstract. Is that sufficient?

Reply:
yes